code/src/components/Styles.js

```diff
@@ -0,0 +1,92 @@
+import styled from 'styled-components/macro';
+
+export const Container = styled.div`
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+    width: 100%;
+`;
+
+export const CardContainer = styled.div`
+    box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2),
+        0px 1px 1px 0px rgba(0, 0, 0, 0.14),
+        0px 1px 3px 0px rgba(0, 0, 0, 0.12);
+    border-radius: .2rem;
+    background: white;
+    margin: 0 auto;
+    margin-bottom: 20px;
+    max-width: 500px;
+    font-family: 'Press Start 2P', cursive;
+    opacity: 0.6;

```

Comment:
Only comment here is to use an RGBA for background instead of a color name/hex and opacity, because it makes the buttons and text and everything inside the CardContainer opaque too - unless that was on purpose of course 😄